docs/apache-airflow/installation.rst

```diff
@@ -29,6 +29,30 @@ installation with other tools as well.
 
     Airflow is also distributed as a Docker image (OCI Image). Consider using it to guarantee that software will always run the same no matter where it is deployed. For more information, see: :doc:`docker-stack:index`.
 
+Version Life Cycle
+'''''''''''''''''
+
+Apache Airflow version life cycle:
+
++---------+---------------------+-----------------+---------------+-----------------+----------------+
+| Version | Current Patch/Minor |      State      | First Release | Limited Support | EOL/Terminated |
++---------+---------------------+-----------------+---------------+-----------------+----------------+
+|    2    |        2.1.0        |    Supported    |  Dec 17, 2020 |     Dec 2021    |       TBD      |
++---------+---------------------+-----------------+---------------+-----------------+----------------+
+|   1.10  |       1.10.15       | Limited Support |  Aug 27, 2018 |     Dec 2020    |    June 2021   |
++---------+---------------------+-----------------+---------------+-----------------+----------------+
+|   1.9   |        1.9.0        |       EOL       |  Jan 03, 2018 |     Aug 2018    |    Aug 2018    |
++---------+---------------------+-----------------+---------------+-----------------+----------------+
+|   1.8   |        1.8.2        |       EOL       |  Mar 19, 2017 |     Jan 2018    |    Jan 2018    |
++---------+---------------------+-----------------+---------------+-----------------+----------------+
+|   1.7   |       1.7.1.2       |       EOL       |  Mar 28, 2016 |     Mar 2017    |    Mar 2017    |
++---------+---------------------+-----------------+---------------+-----------------+----------------+
```

Comment:
Since this is "across" versions, would this possible make more sense on the site and not in the docs, so that old versions don't contain out-of-date info?

Reply:
We don't really have a good "place" for that and since people might get to this page via google, It might make sense here actually, since this is the place that people got when they want to install. 

The only "variable" here is the "TBD" in current version, but unless we get to python-like regular releases, this has to be TBD I think. 

When we update it periodically with releasing new versions, this will stay in the same "stable" link and will be automatically updated. So it might make sense to keep it here. We can also update the TBDs retroactively with already released pages. That will be one simple search replace when we agree to release new major version.

Reply:
The current Patch/Minor is variable too.

Reply:
True. Do we need it there though Maybe we can skip it in the docs? 

Where else could we put it ?

Reply:
> The current Patch/Minor is variable too.

yeah I'm fine with removing it. User can find this information easily in https://pypi.org/project/apache-airflow/#history